src/hooks/sdk/docs/useSelectVideoInputDevice.stories.mdx

```diff
@@ -14,6 +14,11 @@ You can access the available video input device IDs using the `useVideoInputs` h
 (deviceId: string) => Promise<void>;
 ```
 
+Use the returned function from hook only when you want to start a video or the video is already started.
+If used earlier and you select a valid video input with access to your local camera, the webcam LED will turn on.
```

Comment:
Can we say something like "If the video is used earlier" to make it a little clearer?

Reply:
Removed this and provide a general guidance with an updated usage example.